app/models/site_notice.rb

```diff
@@ -72,4 +75,11 @@ def new_consultation_end_date
   def extend_consultation!
     consultation.update!(end_date: new_consultation_end_date)
   end
+
+  def eia_statement
+    eia = planning_application.environment_impact_assessment
+    if eia&.required? && eia&.address_and_fee_present?
+      "<p>View a hard copy of the Environment Statement, free of charge at #{eia.address}. You can buy a hard copy for £#{eia.fee} at the same address</p>"
```

Comment:
Probably not for this PR, but it looks as though the Notify emails/letters for EIA applications may need to use exactly this wording, so it may be useful in the longer term to think about attaching it to the EIA rather than to the site notice so that it can be shared.